dali/base/dadfs.cpp

```diff
@@ -9293,9 +9293,9 @@ class CDFPartFilter : implements IDFPartFilter, public CInterface
                     start = 0;
                 }
                 else if (pn == 0)
-                         throw MakeStringException(0,"Invalid part filter: %s", filter);
-                     else
-                         partincluded[pn-1] = true;
+                        throw MakeStringException(0,"Invalid part filter: %s", filter);
```

Comment:
MakeStringException is considered deprecated, new/changed code should use either makeStringException or makeStringExceptionV in its place.

Reply:
the indentation is off. Make sure spaces vs tabs (vscode setting can ensure inserts spaces vs tabs).

Reply:
trivial/style: space between parameters, i.e. (0, "Invalid...